### DISCUSSION TITLE
为什么在下载页mod会出现这个东西

### DISCUSSION THREAD

**Discussion by HanXiaozeng** [Meloong-Git/PCL#1525] [提问] [answered]
![屏幕截图 2023-03-11 095222](https://user-images.githubusercontent.com/110365422/228385882-2dadddbc-eaf5-4836-a70c-84bc7243d92d.png)
就是这张！我不知道，上面的mod明明都加载出来了呀！

**Comment by Chiloven945** [comment]
你确定你不是眼疾手快截的（
如果有bug请交issue

**Reply by HanXiaozeng** [reply]
额，不是的我眼疾手快截的，他出现了好久，都没有消失！

**Reply by HanXiaozeng** [reply]
而且ta都是在最下面的！

**Comment by HanXiaozeng** [comment]
我现在的pcl一直都是这个界面，我通过修改注册表（差点电脑就坏了）把他调回来了

**Comment by WForst-Breeze** [comment, marked as answer]
请提交综合 Bug 反馈，并提供相关录屏和其他信息以使我们能确认问题存在，谢谢

**Comment by HanXiaozeng** [comment]
抱歉，我只有一张截图，其他的录屏我都没有了！

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows
